lib/components/form/call-taker/date-time-picker.tsx

```diff
@@ -0,0 +1,223 @@
+import { connect } from 'react-redux'
+import { format, toDate } from 'date-fns-tz'
+import { getCurrentTime } from '@opentripplanner/core-utils/lib/time'
+import { IntlShape, useIntl } from 'react-intl'
+import { isMatch, parse } from 'date-fns'
+import { OverlayTrigger, Tooltip } from 'react-bootstrap'
+import coreUtils from '@opentripplanner/core-utils'
+import React, { useEffect, useState } from 'react'
+
+const { getCurrentDate, OTP_API_DATE_FORMAT, OTP_API_TIME_FORMAT } =
+  coreUtils.time
+
+function getDepartureOptions(intl: IntlShape) {
+  return [
+    {
+      // Default option.
+      text: intl.formatMessage({ id: 'components.DateTimeOptions.now' }),
+      value: 'NOW'
+    },
+    {
+      text: intl.formatMessage({ id: 'components.DateTimeOptions.departAt' }),
+      value: 'DEPART'
+    },
+    {
+      text: intl.formatMessage({ id: 'components.DateTimeOptions.arriveBy' }),
+      value: 'ARRIVE'
+    }
+  ]
+}
+
+/**
+ * Time formats passed to date-fns to parse the user's time input.
+ */
+const SUPPORTED_TIME_FORMATS = [
+  'h:mmaaaaa',
+  'hmmaaaaa',
+  'haaaaa',
+  'Hmm',
+  'Hm',
+  'H:mm',
+  'H:m',
+  'H',
+  'HH:mm'
+]
```

Comment:
Is there a reason why you're only using 'a' and 'p' here as supported formats? It's definitely faster to type "5p" instead of needing to type "5pm", but it does feel odd for '5pm' to throw an error

Reply:
Good catch thanks! Totally just forgot it. In there now